packages/cli/src/lib/vite/mini-oxygen.ts

```diff
@@ -93,6 +64,8 @@ export async function startMiniOxygenRuntime({
         name: 'setup-environment',
         modules: true,
         scriptPath,
+        compatibilityDate: undefined,
+        compatibilityFlags: undefined,
```

Comment:
MiniOxygen adds compat data and flags automatically to every worker. We need to overwrite them here because this worker is a special one that is only used as service binding.